src/main/java/uk/gov/hmcts/reform/em/hrs/service/ccd/CaseDataContentCreator.java

```diff
@@ -22,13 +24,19 @@ public class CaseDataContentCreator {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CaseDataContentCreator.class);
 
+    private static final String TTL_SUSPENDED = "No";
```

Comment:
Following the convention, could we please rename the Constant to NO ?

Reply:
do you mean the value or variable name?

Reply:
variable name - TTL_SUSPENDED to NO

Reply:
this does not make sense, we can not understand what it is for?

Reply:
rename to TTL_SUSPENDED_NO